docs/src/content/docs/it/guides/sidebar.mdx

```diff
@@ -397,20 +423,20 @@ La navigazione della documentazione in spagnolo genererà la seguente barra late
 <SidebarPreview
```

Comment:
I cannot properly reference it, but just above this example, the sentence should reference Brazilian Portuguese as the example language was updated.